src/circllhist.h

```diff
@@ -33,7 +33,7 @@
 #define CIRCLLHIST_H
 
 #ifdef __cplusplus
-extern "C" {
+extern "C" { #FFI_SKIP
```

Comment:
This isn't a valid CPP directive and shouldn't be in an h file.

Reply:
This is a valid C comment, and is intended as a hint to the generation script.
We could add a space to make clear it is not a directive.

Reply:
My bad. Changing this. 1 sec!

Reply:
Since when is # a valid C comment?

Reply:
yes. Sorry, this should be `//FFI_SKIP`. Changing this now.